### PR TITLE
data organization, preparation and annotation

### PR DESCRIPTION
**Pull request by Fatimat01**
- running the code will create the directories and cleanup existing ones
- dynamically split data into train, test and valid in the right proportion
- annotate the data and create labels for corresponding images

### CODE REVIEW

File: code.ipynb
Comment (victorhg) [inline code.ipynb:1]: Can’t review the code - only available raw metadata. Accepting

**Review by victorhg — APPROVED** [review]

**Comment by victorhg** [comment]
Only suggestion would be to rename `code.ipynb` to a name with explicit explanation - `data_processing`, `data_setup` or something like that

**Review by caliskate — CHANGES_REQUESTED** [review]
the construction or custom dataset does not appear to have a label for goggles, requesting to add class "Person", "Mask" ,"NO-Mask", "NO-Safety Vest", "NO-Hardhat"

**Review by caliskate — APPROVED** [review]
approving to review, will likely need to update class names

**Pull request merged**